README.md

```diff
@@ -50,6 +50,8 @@ For more details how to set up Visual Studio Code, read [this document](https://
 1. Go into project folder
 2. run `npm run test`
 
+To run a single test, use `npm run mocha-single <path/to/unit.test.js>`.
```

Comment:
I think this needs to be slightly adjusted:

```suggestion
To run a single test, use `npm run mocha-single -- <path/to/unit.test.js>`.
```

— https://docs.npmjs.com/cli/run-script#description

Reply:
🤔 Interesting. It works without the `--`. I'll add it to the recommendation nevertheless, to make it more explicit.